tmt/steps/provision/mrack.py

```diff
@@ -1097,6 +1170,10 @@ class CreateJobParameters:
     beaker_job_owner: Optional[str]
     public_key: list[str]
     group: Optional[str]
+    auth_dict: Optional[str]
```

Comment:
I really despise when type is encoded in the variable name. Especially when it's incorrect. Is it a dictionary or a string?

Reply:
`bootc_credentials`, please - `auth` does not say anything about what it's used for, and `dist` is simply wrong.